tests/data/generator/t_protocol_no_ct/protocol.yaml

```diff
@@ -11,7 +11,7 @@ fingerprint:
   message.py: QmTVfQQG8fjXi28qUQgVW3aaaMeam4mJthbjNDT54Jnctc
   serialization.py: QmPX4KzaEfK9JwbbHAmmB1rZp1guJTW5PgPan7ZGAp36DH
   t_protocol_no_ct.proto: Qmc8KkKnWZ9utBxrbEyWhVDRdut87DkFvmHP3SYUg4J3EU
```

Comment:
You're using wrong protoc version:
```
protoc --version
libprotoc 3.11.4
```

Reply:
`python scripts/generate_all_protocols.py`

Reply:
done